tensorflow/lite/java/BUILD

```diff
@@ -13,7 +13,9 @@ load("//tensorflow/lite/java:aar_with_jni.bzl", "aar_with_jni")
 
 JAVA_SRCS = glob([
     "src/main/java/org/tensorflow/lite/*.java",
-])
+]) + [
+   "//tensorflow/lite/delegates/nnapi/java/src/main/java/org/tensorflow/lite/nnapi:nnapi_delegate"
```

Comment:
I'm wondering if we should only add the NNAPI delegate for Android?

Reply:
use `//tensorflow:android` to check if it's building for Android